asv.conf.json

```diff
@@ -0,0 +1,160 @@
+{
+    // The version of the config file format.  Do not change, unless
```

Comment:
In general, I would remove all the instructive comments from the JSON here and link to the original instructions JSON for reference. Especially the big unused, commented out sample JSON sections. 

I'm not against comments (even though they look terrible on Github :)) - if we want to add Cirq specific comments we are welcome to, which I think will be pretty minimal at this point.

Reply:
I personally would have preferred to keep the comments in the file so that the next time we have to edit it, the comments come in handy. 
Anyways, removed them.